### PR TITLE
/tg/ rebase work

### PR DESCRIPTION
**Pull request by SweptWasTaken**
## About The Pull Request

I just dumped the voidcrew stuff in and am going to go down the list fixing bugs and pathing and what-not.

### CODE REVIEW

File: code/__DEFINES/__VOID_DEFINES/mobs.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//Defines for species IDs
+#define SPECIES_IPC "ipc"
+#define SPECIES_KEPORI "kepori"
+
+//Species bitflags for sprite sheets.
+#define FLAG_IPC (1<<9)
+
+//Bodytype defines for how things can be worn.
+#define BODYTYPE_KEPORI (1<<5) //Just Kepori
```
Comment (Kapu1178) [inline code/__DEFINES/__VOID_DEFINES/mobs.dm]: why the FUCK would you modularize BITFLAGS?

Comment (SuperNovaa41) [reply code/__DEFINES/__VOID_DEFINES/mobs.dm]: because it literally doesn't matter? lmao, all it needs is better documentation

Comment (ShibaInuLord) [reply code/__DEFINES/__VOID_DEFINES/mobs.dm]: why wouldn't you though

Comment (Kapu1178) [reply code/__DEFINES/__VOID_DEFINES/mobs.dm]: Have fun when someone doesn't realise 1<<5 is an already used bodytype.

also, the entirety of this bitfield sequence is a Kapulimbs feature anyway, which is something TG doesn't have at all.

Comment (SuperNovaa41) [reply code/__DEFINES/__VOID_DEFINES/mobs.dm]: > Have fun when someone doesn't realise 1<<5 is an already used bodytype.

two second fix, genuinely a non issue
> also, the entirety of this bitfield sequence is a Kapulimbs feature anyway, which is something TG doesn't have at all.

awesome ill be glad to get rid of it then

Comment (SweptWasTaken) [reply code/__DEFINES/__VOID_DEFINES/mobs.dm]: > why the FUCK would you modularize BITFLAGS?

Ask @SuperNovaa41 this, i'm just fixin' errors.

**Pull request closed without merging**